docker-compose.yml

```diff
@@ -23,3 +23,5 @@ services:
     volumes:
        - ./apps:/opt/spark-apps
        - ./data:/opt/spark-data
+    ports:
+      - "9091:8081" 
```

Comment:
this additional port would affect scaling the workers as the port will clash best its removed

Reply:
I'm fix it.